server/src/test/java/org/opensearch/extensions/action/ExtensionsActionTests.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.extensions.action;
+
+import org.opensearch.action.ActionFuture;
+import org.opensearch.test.OpenSearchSingleNodeTestCase;
+
+import java.util.concurrent.ExecutionException;
+
+public class ExtensionsActionTests extends OpenSearchSingleNodeTestCase {
+
+    public void testExtensionAction() throws ExecutionException, InterruptedException {
```

Comment:
usually tests contain assertions of some sort. :)

Reply:
Yup I just realized I missed a commit :D